src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -4,6 +4,7 @@ const ProjectItemWrapper = styled.section`
 		text-align: left;
 		align-self: flex-start;
 		margin-left: -5rem;
+		margin-top: 5rem;
```

Comment:
@Himanshu664, I'm hoping that we can leave this at the same height (keep the sections relatively close to one another), but push this header to the left of the page instead.

Reply:
Sure Sir🙏

Reply:
Immediately, though, let's accept your PR as it fixes the overlap. This will buy you some time to refactoring into a left-aligned solution, which might mean that you have to modify other sections/pages/modules.

Reply:
Or not... I seen now that this update doesn't fix the overlap on smaller screens. We do want a solution quickly given that this is on the home page.

Reply:
Commited new changes please have a look

Reply:
@leecalcote Added Media Queries for all the screens please check